tests/test_command_check_cloudformation.py

```diff
@@ -1,103 +1,100 @@
+def test_check_cloudformation_with_no_args_summarises_all_failures(
+    test_application, copilot_directory: Path
+) -> None:
+    prepare_fake_cloudformation_templates(copilot_directory, passing=False)
 
+    result = CliRunner().invoke(
+        check_cloudformation_command, args=["--directory", copilot_directory]
+    )
 
-def prepare_fake_cloudformation_templates(copilot_directory: Path, passing: str) -> None:
-    template = (
-        "valid_cloudformation_template.yml" if passing else "invalid_cloudformation_template.yml"
+    assert (
+        "The CloudFormation templates failed the following checks:\n  - lint [E0000 could not find expected ':'"
+        in result.output
     )
-    ensure_directory_does_not_exist(copilot_directory)
-    addons_directory = Path(f"{BASE_DIR}/tests/test-application/copilot/environments/addons")
-    addons_directory.mkdir(parents=True, exist_ok=True)
-    copyfile(f"{BASE_DIR}/tests/fixtures/{template}", f"{addons_directory}/{template}")
 
 
-@patch("dbt_copilot_helper.commands.check_cloudformation.prepare_cloudformation_templates")
-def test_outputs_passed_results_summary(
-    patched_prepare_cloudformation_templates, copilot_directory: Path
-) -> None:
-    patched_prepare_cloudformation_templates.return_value(None)
+def test_linting_check_passed(test_application, copilot_directory: Path) -> None:
     prepare_fake_cloudformation_templates(copilot_directory, passing=True)
 
-    result = CliRunner().invoke(check_cloudformation_command)
+    result = CliRunner().invoke(
+        check_cloudformation_command, args=["lint", "--directory", copilot_directory]
+    )
 
-    assert (
-        "The CloudFormation templates passed the following checks :-)\n  - lint" in result.output
-    ), "The passed checks summary was not outputted"
+    assert result.exit_code == 0
+    assert "The CloudFormation templates passed the following checks:\n  - lint" in result.output
 
 
-@patch("dbt_copilot_helper.commands.check_cloudformation.prepare_cloudformation_templates")
-def test_outputs_failed_results_summary(
-    patched_prepare_cloudformation_templates, copilot_directory: Path
-) -> None:
-    patched_prepare_cloudformation_templates.return_value(None)
+def test_linting_check_failed(test_application, copilot_directory: Path) -> None:
     prepare_fake_cloudformation_templates(copilot_directory, passing=False)
 
-    result = CliRunner().invoke(check_cloudformation_command)
+    result = CliRunner().invoke(
+        check_cloudformation_command, args=["lint", "--directory", copilot_directory]
+    )
 
+    assert result.exit_code != 0
     assert (
-        "The CloudFormation templates failed the following checks :-(\n  - lint\n" in result.output
-    ), "The failed checks summary was not outputted"
+        "The CloudFormation templates failed the following checks:\n  - lint [E0000 could not find expected ':'"
+        in result.output
+    )
+
+    # assert "E0000 could not find expected ':'" in result.output
```

Comment:
Looks like this can be removed.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/173/commits/0d511cd7a0a5feb7634d37125981da6e2b6ecd28.